sklearn/neural_network/tests/test_mlp.py

```diff
@@ -437,16 +449,38 @@ def test_predict_proba_multi():
     assert_array_equal(y_log_proba, np.log(y_proba))
 
 
+def test_predict_proba_multilabel():
+    # Test that predict_proba works as expected for multilabel.
+    # Multilabel should not use softmax which makes probabilities sum to 1
+    X, y = make_multilabel_classification(n_samples=50, random_state=0,
+                                          return_indicator=True)
```

Comment:
Since y is a 2D indicator array, the convention would be to use the uppercase variable name `Y` instead.